content/en/platform/corda/5.0-dev-preview-1/tutorials/building-cordapp/c5-basic-cordapp-flows.md

```diff
@@ -543,10 +565,12 @@ Finish the initiating flow by continuing with the same steps you followed when c
 2. Build the transaction using the `TransactionBuilderFactory` service.
 3. Verify that the transaction is valid with the `txBuilder.verify` method.
 4. Sign the transaction using the `txBuilder.sign` method.
-5. Send the state to the counterparty using the `flowMessaging` service, then receive it using the `FlowEngine` and the subflow `CollectSignaturesFlow`.
-6. Notarize and record the transaction in both parties' vaults using the `FlowEngine` service and the subflow `FinalityFlow`.
+5. Send the state to Peter using the `flowMessaging` service, then receive it using the `FlowEngine` and the subflow `CollectSignaturesFlow`.
+6. Get the transaction notarized in both parties' vaults using the `FlowEngine` service and the subflow `FinalityFlow`.
+   After the transaction is performed, the issuer will have the consumed state in their vault and they won't have a reference to the new state. The new state generated by this transaction will be stored in the counterparty's vault.
 7. Return a `SignedTransactionDigest`.
```

Comment:
@peterli-r3 could you please explain a bit more why (apart from the fact that once you sign the transaction you can't go back) it's such an important step?

Reply:
which step is an such important step?

Reply:
added a note and resolved it myself